intercom-java/src/main/java/io/intercom/api/Company.java

```diff
@@ -262,6 +304,15 @@ public Company setRemoteCreatedAt(long remoteCreatedAt) {
         return this;
     }
 
+    public long getLastRequestAt() {
```

Comment:
Should we set the entities lastRequestAt as well? As we do with the other API attributes?

Reply:
@jonnyom not sure I get that 😅 Isn't this the `setLastRequestAt` below this what you're referring to? Or maybe I'm missing something